apps/omg_watcher_security/lib/omg/watcher_security/sync_supervisor.ex

```diff
@@ -51,81 +51,60 @@ defmodule OMG.Watcher.SyncSupervisor do
         get_events_callback: &Eth.RootChain.get_deposits/2,
         process_events_callback: &OMG.State.deposit/1
       ),
-      # this instance of the listener sends deposits to be consumed by the convenience API
-      EthereumEventListener.prepare_child(
-        service_name: :convenience_deposit_processor,
-        synced_height_update_key: :last_convenience_deposit_processor_eth_height,
-        get_events_callback: &Eth.RootChain.get_deposits/2,
-        process_events_callback: fn deposits ->
-          Watcher.DB.EthEvent.insert_deposits!(deposits)
-          {:ok, []}
-        end
-      ),
```

Comment:
Double checking. This is removed because it's being taken care of in https://github.com/omisego/elixir-omg/pull/1096/files#diff-185b2a99d2d25f3ca411079d17a1c8d6R121-R126  (also `: convenience_exit_processor`)?